.changeset/thick-boxes-fail.md

```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/starlight": patch
+---
+
+Make aside's title can be translated.
```

Comment:
Because this adds a feature, we can use `minor` for the changeset and I might suggest this as a description:

```suggestion
---
"@astrojs/starlight": minor
---

Add i18n support for default aside labels
```